include/small/Conv2DLayer.hpp

```diff
@@ -190,10 +190,12 @@ class Conv2DLayer : public Layer<BufferT>
         }
         else if (m_activation_type == LEAKY)
         {
+            BufferT neg_slope(1);
```

Comment:
Change m_leaky_slope to a BufferT and initialize it in the constructor rather than constructing on here each time.